website/src/components/icons-list.tsx

```diff
@@ -72,11 +74,28 @@ const IconsList = () => {
     >
       {Object.keys(iconList).map((key, i) => {
         const Icon = iconList[key]
+        const { onCopy } = useClipboard(key)
+
+        const onCopyIcon = () => {
+          onCopy()
+
+          toast({
+            title: `'${key}' copied to clipboard`,
+            status: "success",
+            duration: 2000,
+            isClosable: false,
+          })
+        }
+
         return (
           <Flex
             p={3}
+            onClick={() => onCopyIcon()}
             key={i}
             align="center"
+            cursor="pointer"
+            transition="all 0.2s ease"
+            _hover={{ opacity: 0.7 }}
```

Comment:
Maybe I should remove this hover effect, unsure

Reply:
The current implementation is not very accessible. Let me point out a few things and how to fix them:

The `Flex` component renders a `<div>`.
A div has no meaning in terms of accessibility, but we want to make this one interactive.

There are two ways I know of:

1. Adding the correct aria-labels like `role="button"` and implementing keyboad navigation to handle focus `tabIndex={0}` and interaction (Did you notice that buttons fire their onClick handler when pressing "enter"?)
2. Using the html element `button` and do not use `p` tags inside of it. This lets the browser do all the magic mentioned above.

The second one sound quite simpler, doesn't it 😄 

```jsx
<Button // button
  onClick={() => console.log("clicked")}
  variant="outline"
  p="3"
  height="auto"
  align="center"
  flexDir="column"
  justify="center"
>
  <CopyIcon />
  <Text 
    as="span" // <-- no `<p>` tag
    mt={2} 
    fontSize="sm" 
    textAlign="center">
    CopyIcon
  </Text>
</Button>
```